src/components/Charts/zoneUtils.js

```diff
@@ -0,0 +1,145 @@
+import {
+  baseOptions,
+  formatDecimal,
+  formatPercent,
+  getMaxY,
+  getTickPositions,
+  last,
+  currentValueAnnotation,
+  parseDate,
+  roundAxisLimits,
+  zoneAnnotations,
+} from './utils';
+import {
+  CASE_GROWTH_RATE,
+  HOSPITAL_USAGE,
+  POSITIVE_TESTS,
+} from '../../enums/zones';
+
+export { parseDate };
+
+const toHighchartZone = (zone, level) => ({
+  color: zone.color,
+  name: zone.label,
+  value: isFinite(zone.upperLimit) ? zone.upperLimit : undefined,
+  className: 'ZoneChart__Line',
+  labelClassName: `ZoneAnnotation ZoneAnnotation--${level}`,
+});
+
+const ZONES_RT = [
+  toHighchartZone(CASE_GROWTH_RATE.LOW, 'Low'),
+  toHighchartZone(CASE_GROWTH_RATE.MEDIUM, 'Medium'),
+  toHighchartZone(CASE_GROWTH_RATE.HIGH, 'High'),
+];
+
+const ZONES_POSITIVE_RATE = [
+  toHighchartZone(POSITIVE_TESTS.LOW, 'Low'),
+  toHighchartZone(POSITIVE_TESTS.MEDIUM, 'Medium'),
+  toHighchartZone(POSITIVE_TESTS.HIGH, 'High'),
+];
+
+const ZONES_HOSPITAL_USAGE = [
+  toHighchartZone(HOSPITAL_USAGE.LOW, 'Low'),
+  toHighchartZone(HOSPITAL_USAGE.MEDIUM, 'Medium'),
+  toHighchartZone(HOSPITAL_USAGE.HIGH, 'High'),
+];
+
+export const optionsRt = (data, endDate) => {
+  const { x, y } = last(data);
+  const [minYAxis, maxYAxis] = roundAxisLimits(0, getMaxY(data));
+  return {
+    ...baseOptions,
+    xAxis: {
+      ...baseOptions.xAxis,
+      max: parseDate(endDate),
+    },
+    yAxis: {
+      ...baseOptions.yAxis,
+      tickPositions: getTickPositions(minYAxis, maxYAxis, ZONES_RT),
+    },
+    series: [
+      {
+        className: 'ZoneChart__AreaRange',
+        type: 'arearange',
+        fillColor: '#efefef',
+        zIndex: 0,
+        enableMouseTracking: false,
+        data: data.map(d => [d.x, d.low, d.hi]),
+      },
+      {
+        name: 'Rt',
+        type: 'spline',
+        zones: ZONES_RT,
+        data,
+      },
+    ],
+    annotations: [
+      currentValueAnnotation(x, y, formatDecimal(y)),
+      ...zoneAnnotations(endDate, minYAxis, maxYAxis, y, ZONES_RT),
+    ],
+  };
+};
+
+export const optionsPositiveTests = (data, endDate) => {
+  const { x, y } = last(data);
+  const [minYAxis, maxYAxis] = roundAxisLimits(0, getMaxY(data));
+  return {
+    ...baseOptions,
+    xAxis: {
+      ...baseOptions.xAxis,
+      max: parseDate(endDate),
+    },
+    yAxis: {
+      ...baseOptions.yAxis,
+      labels: {
+        formatter: function () {
+          return formatPercent(this.value);
+        },
+      },
+      tickPositions: getTickPositions(minYAxis, maxYAxis, ZONES_POSITIVE_RATE),
+    },
+    annotations: [
+      currentValueAnnotation(x, y, formatPercent(y)),
+      ...zoneAnnotations(endDate, minYAxis, maxYAxis, y, ZONES_POSITIVE_RATE),
+    ],
+    series: [
+      {
+        name: 'Positive Tests',
```

Comment:
eventually think this should be a constant somewhere as well

Reply:
yes, let's fix it once we centralize the enums - this is only used for the tooltips (which need fixing too)